tox.ini

```diff
@@ -1,28 +1,40 @@
 [tox]
-envlist=pep8,isort,py{35}-django{19,18}-cms{32,31}, py{34,33,27}-django{19,18,17}-cms{32,31}, py{34,33,27,26}-django16-cms{32,31}
+envlist=
+    flake8
+    isort
+    py35-django19-cms32
+    py35-django18-cms{32,31}
+    py{34,33,27}-django19-cms32
+    py{34,33,27}-django{18,17}-cms{32,31}
+    py{34,33,27}-django17-cms30
+    py{34,33,27,26}-django16-cms{32,31,30}
 skip_missing_interpreters=True
 
 [testenv]
 commands = {env:COMMAND:python} test_settings.py
 deps=
     py26: unittest2
-    django16: django<1.7
-    django17: django<1.8
-    django18: django<1.9
-    django19: django==1.9.a1
-    django-parler>=1.4,<1.5
+    django16: django>=1.6,<1.7
+    django17: django>=1.7,<1.8
+    django18: django>=1.8,<1.9
+    django19: django>=1.9.1,<1.10
+    django-parler>=1.4
     django-appdata>=0.1.4
     https://github.com/nephila/djangocms-helper/archive/develop.zip
     cms30: https://github.com/divio/django-cms/archive/support/3.0.x.zip
     cms31: https://github.com/divio/django-cms/archive/support/3.1.x.zip
-    cms32: https://github.com/divio/django-cms/archive/develop.zip
+    cms32: https://github.com/divio/django-cms/archive/release/3.2.x.zip
     -r{toxinidir}/requirements-test.txt
 
-[testenv:pep8]
+[testenv:flake8]
 deps = flake8
 commands = flake8
 skip_install = true
 
+[flake8]
```

Comment:
If that change is not critical I'd prefer to use the same approach as on essential add-ons.
Also I've encountered a bug where flake uses any ``setup.cfg`` which is accessible through ``PATH`` (i.e. from home directory)